CAIPs/caip-104.md

```diff
@@ -0,0 +1,229 @@
+
+`* review-period-end:` <date review period ends>
+
+` type:` <Standards Track (Core, Networking, Interface, ERC)  | Informational | Meta>
+
+` * category:` <Core | Networking | Interface | ERC>
+
+` created:` <date created on>
+
+` * updated:` <comma separated list of dates>
+
+` * requires:` <CAIP number(s)>
+
+` * replaces:` <CAIP number(s)>
+
+` * superseded-by:` <CAIP number(s)>
+
+#### `title` header
+
+{string1} should be the natural-language spelling/name of the namespace, e.g.
+"EIP-155" for `eip155`; if this name is technical or otherwise counterintuitive,
+an optional clarification may be affixed, e.g. ", aka EVM Chains." If the
+reference in question is a per-CAIP reference, a common name for the specified
+referent can be added for further clarity, i.e. " - Assets" (for CAIP-19) or " -
+Addresses" (for CAIP-10), as these markdown files will double as page-titles if
+rendered in a browser.
+
+#### `author` header
+
+The `author` header optionally lists the names, email addresses or usernames of
+the authors/owners of the CAIP. Those who prefer anonymity may use a username
+only, or a first name and a username. The format of the author header value must
+be:
+
+> Random J. User &lt;address@dom.ain&gt;
+
+or
+
+> Random J. User (@username)
+
+if the email address or GitHub username is included, and
+
+> Random J. User
+
+if the email address is not given.
+
+#### `resolution` header
+
+If ratification of this document was recorded at a permanent URL (e.g. the
+recorded minutes of a CASA meeting or mailing list), that URL can be placed here
+for additional context.
+
+#### `discussions-to` header
+
+While an CAIP is a draft, a `discussions-to` header will indicate the mailing
+list or URL where the CAIP is being discussed.
+
+#### `type` header
+
+The `type` header specifies the type of CAIP: Standards Track, Meta, or
+Informational.
+
+#### `created` header
+
+The `created` header records the date that the CAIP was assigned a number. Both
+headers should be in yyyy-mm-dd format, e.g. 2001-08-14.
+
+#### `updated` header
+
+The `updated` header records the date(s) when the CAIP was updated with
+"substantial" changes. This header is only valid for CAIPs of Draft and Active
+status.
+
+#### `requires` header
+
+CAIPs may have a `requires` header, indicating the CAIP numbers that this
+reference depends on. 
+
+#### `superseded-by` and `replaces` headers
+
+CAIPs may also have a `superseded-by` header indicating that an CAIP has been
+rendered obsolete by a later document; the value is the number of the CAIP that
+replaces the current document. The newer CAIP must have a `replaces` header
+containing the number of the CAIP that it rendered obsolete.
+
+## Auxiliary Files
+
+CAIPs may include auxiliary files such as diagrams. Such files must be named
+CAIP-XXXX-Y.ext, where “XXXX” is the CAIP number, “Y” is a serial number
+(starting at 1), and “ext” is replaced by the actual file extension (e.g.
+“png”).
```

Comment:
this IMHO contradicts the relative assets before. I actually like this one more - maybe we should remove the relative mention before

Reply:
good catch! done.